### PR TITLE
ER-86-API-Type-Addition

### PR DESCRIPTION
**Pull request by amansiid**
Finished API Type Addition requirements:
Endpoint URL => /api/inventory/type-add

### CODE REVIEW

**Review by tranleduyan — APPROVED** [review]
I cleaned up some code and typo! Otherwise, the code is solid! Good work, bro!

File: src_code/express_app/app.js
Comment (mohammedking11-spu) [inline src_code/express_app/app.js:1]: Good use of comments, good job!

File: src_code/src/controllers/interfaces/IInventory.js
Comment (mohammedking11-spu) [inline src_code/src/controllers/interfaces/IInventory.js:1]: Good practice of proper modules initializing variables as well as exports.

File: src_code/src/controllers/services/Inventory/EquipmentRemoval.js
Comment (mohammedking11-spu) [inline src_code/src/controllers/services/Inventory/EquipmentRemoval.js:1]: Proper way of explaining functions with clarifying important parameters and return types.

File: src_code/src/controllers/services/Inventory/ModelAddition.js
Comment (mohammedking11-spu) [inline src_code/src/controllers/services/Inventory/ModelAddition.js:1]: Proper way of describing functions thoroughly, much appreciated, it also helps the reader with knowing exactly what those functions do/return.

File: src_code/src/controllers/services/Inventory/TypeAddition.js
Comment (mohammedking11-spu) [inline src_code/src/controllers/services/Inventory/TypeAddition.js:1]: Very nicely written functions for both TypeAddition and TypeValidationAddition, and being consistent with the style of writing code is the key, proper way of using comments, good job!

File: src_code/src/routes/InventoryRoutes.js
Comment (mohammedking11-spu) [inline src_code/src/routes/InventoryRoutes.js:1]: Very nicely written sustainable code with proper use of error handling and proper comments.

File: src_code/src/utils/interfaces/IDBHelperFunctions.js
Comment (mohammedking11-spu) [inline src_code/src/utils/interfaces/IDBHelperFunctions.js:1]: Staying consistent with variable naming conventions.

File: src_code/src/utils/services/helper_functions/db_helper_functions/user/UserDBHelperFunctions.js
Comment (mohammedking11-spu) [inline src_code/src/utils/services/helper_functions/db_helper_functions/user/UserDBHelperFunctions.js:1]: Both functions, GetTypeInfoByName and AddTypeToDatabase, looks written in a clean way with proper use of error handling, try and catch, with proper use of comments to explain them thoroughly.

**Review by mohammedking11-spu — APPROVED** [review]
Everything looks clear after Le's changes, great job on using a lot of comments to explain every function in the proper way. I also liked the try/catch error handling that you used, very effective especially for our application. Great job.

**Pull request merged**